### PR TITLE
Replace 'Drain' with 'Change' for clearer terminology

### PR DESCRIPTION
**Pull request by BEULAHEVANJALIN**
#52

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -47,20 +47,20 @@ pub struct CoinSelectionOpt {
...
     /// Estimate of cost of spending the output.
     pub cost_per_output: u64,
 
-    /// Minimum value allowed for a drain (change) output.
-    pub min_drain_value: u64,
+    /// Minimum value allowed for a change output.
+    pub min_change_value: u64,
 
     /// Strategy to use the excess value other than fee and target
     pub excess_strategy: ExcessStrategy,
```
Comment (jkciw) [inline src/lib.rs:66]: #54 
Please replace `ToDrain` in excess strategy with `ToChange` 
```rust
pub enum ExcessStrategy {
    ToFee,
    ToRecipient,
    ToDrain,
}
```

File: src/lib.rs [hunk 2]
```diff
@@ -672,7 +672,7 @@ fn calculate_waste(
         waste += (accumulated_value - (options.target_value + estimated_fee));
```
Comment (jkciw) [inline src/lib.rs:672]: #54 Please change this reference to ToDrain 
```rust
if options.excess_strategy != ExcessStrategy::ToDrain {
        // Change is not created if excess strategy is ToFee or ToRecipient. Hence cost of change is added
        waste += (accumulated_value - (options.target_value + estimated_fee));
    } else {
```

File: src/lib.rs [hunk 3]
```diff
@@ -852,30 +852,30 @@ mod test {
...
-            drain_weight: 50,
-            drain_cost: 10,
+            change_weight: 50,
+            change_cost: 10,
             cost_per_input: 20,
             cost_per_output: 10,
-            min_drain_value: 500,
+            min_change_value: 500,
             excess_strategy: ExcessStrategy::ToDrain,
         }
```
Comment (jkciw) [inline src/lib.rs:861]: / #54 Please change this reference to ToDrain

File: src/lib.rs [hunk 4]
```diff
@@ -852,30 +852,30 @@ mod test {
...
             base_weight,
-            drain_weight: 50,
-            drain_cost: 10,
+            change_weight: 50,
+            change_cost: 10,
             cost_per_input: 20,
             cost_per_output: 10,
-            min_drain_value,
+            min_change_value,
             excess_strategy: ExcessStrategy::ToDrain,
```
Comment (jkciw) [inline src/lib.rs:879]: #54 Please change reference to ToDrain

File: src/lib.rs [hunk 5]
```diff
@@ -927,11 +927,11 @@ mod test {
...
             base_weight: 10,
-            drain_weight: 50,
-            drain_cost: 10,
+            change_weight: 50,
+            change_cost: 10,
             cost_per_input: 20,
             cost_per_output: 10,
-            min_drain_value: 500,
+            min_change_value: 500,
             excess_strategy: ExcessStrategy::ToDrain,
```
Comment (jkciw) [inline src/lib.rs:935]: #54 Please change ToDrain

Comment (BEULAHEVANJALIN) [reply src/lib.rs]: Thank you for the feedback!

File: src/lib.rs (hunk 1 above)
Comment (BEULAHEVANJALIN) [reply src/lib.rs:66]: Done.

**Pull request merged**